numba/cuda/compiler.py

```diff
@@ -823,6 +824,17 @@ def __init__(self, dispatcher, griddim, blockdim, stream, sharedmem):
         self.stream = stream
         self.sharedmem = sharedmem
 
+        if config.CUDA_LOW_OCCUPANCY_WARNINGS:
+            ctx = get_context()
+            smcount = ctx.device.MULTIPROCESSOR_COUNT
+            grid_size = griddim[0] * griddim[1] * griddim[2]
+            if grid_size < 2 * smcount:
+                msg = ("Grid size ({grid}) < 2 * SM count ({sm}) "
+                       "will likely result in GPU under utilization due"
+                       "to low occupancy.")
```

Comment:
I just noticed there's a space missing here so it ends up printing "dueto low occupancy."

```suggestion
                       "will likely result in GPU under utilization due "
                       "to low occupancy.")
```